eng/mgmt/mgmtmetadata/azure-kusto_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azure-kusto/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\Source\azure-sdk-for-net\sdk
-2020-07-13 11:49:52 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azure-kusto/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\Users\astauben\Desktop\c\azure-sdk-for-net\sdk
```

Comment:
?

Reply:
Autogenerated. I've looked at the file's history and it's looks ok (both Lucy and Zohar made similar changes)

Reply:
How did you auto-generate this because your username "C:\Users\astauben\Desktop\" shouldn't be in the file path? Thanks